dist/index.js

```diff
@@ -56,7 +56,7 @@ var LicenseWatch = function (_EventEmitter) {
   _createClass(LicenseWatch, [{
     key: 'fetch',
     value: function () {
-      var _ref = _asyncToGenerator( /*#__PURE__*/regeneratorRuntime.mark(function _callee() {
+      var _ref = _asyncToGenerator(regeneratorRuntime.mark(function _callee() {
```

Comment:
why this change happens?

Reply:
no, because we chose to ship it with the transpiled version rather than do it on the user's end install.